### DISCUSSION TITLE
HTTP / HTTPS issue with Livewire

### DISCUSSION THREAD

**Discussion by ikeThun** [livewire/livewire#7483] [1. Help]
Hello
I'm running a livewire application with Volt and it seems like livewire thinks my base_url is http, even tho it's set to https in the .env file. I'm enforcing https in every file possible. This way my file uploads don't work, and die with error: 401.

I also get errors like` You can't use wire:navigate with a link to a different root domain: http://test.laravelapp.hu/form` when i'm on https://test.laravelapp.hu/form
and
DOMException: The operation is insecure.
in console.

I have tried allowing trusted proxies, setting different hosts, forcing the reverse proxy to force https

Any help appreciated,
Thanks you in advance.

**Comment by pronode** [comment]
Cleared config with `php artisan config:clear`?

**Reply by ikeThun** [reply]
yes, sadly, same goes for every command i could think of that helps with clearing cache :\ but thank you

**Comment by ArtMin96** [comment]
```php
if ($this->app->environment('production')) {
    URL::forceScheme('https');
}
```

**Reply by ikeThun** [reply]
I have this in my AppServiceProvider, but no luck, but thank you

**Comment by pronode** [comment]
Do you also encounter this problem when using "relative" urls like `<a href="/form" ...>`?

**Reply by ikeThun** [reply]
I only use relative urls, with wire:navigate

**Comment by pronode** [comment]
Did you try to hard-refresh browser cache? This may be a silly advice but trying to help.

**Comment by kominfosekadau** [comment]
I have same problem here in live server but in local dev this not happens

**Comment by xflysaround** [comment]
Had the same issue. Fixed it by updating the asset_url and app_url in config/livewire.php. Both need to point to a 'https' url in production.

**Comment by Fulue** [comment]
Use the tag `<meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">`
